doc/tutorial.txt

```diff
@@ -489,6 +489,10 @@ taskB ...-------------------|resume taskA|run 10    |--------------------
 * yield: String. Calls pthread_yield(), freeing the CPU for other tasks. This has a
 special meaning for SCHED_DEADLINE tasks. String can be empty.
 
+* fork: String. Instead of creating all tasks at start time, you can fork one
+at any point of time using this event. The name of the forked task must match
+one of the defined tasks.
+
```

Comment:
you have to update the description of "instance". Add that 0 will now parse and create data structure of the task but will not create any pthread.

Reply:
Added that to the instance parameter description and folded it into the (Parse tasks with 0 instance value) commit